lib/kerosene.ex

```diff
@@ -47,16 +47,21 @@ defmodule Kerosene do
   defp get_total_count(_count, repo, query) do
     primary_key = get_primary_key(query)
 
-    total_pages =
+    total_count =
       query
       |> exclude(:preload)
       |> exclude(:order_by)
       |> exclude(:select)
+      |> consider_group_by()
```

Comment:
No, because the number of rows depends on it. For example we have 2 products and 3 orders for each. If we exclude grouping by products there will be 6 rows after join.